pkg/api/analysisrun/result.go

```diff
@@ -14,6 +14,7 @@ type (
 type DynamicAnalysisResults struct {
 	StraceSummary DynamicAnalysisStraceSummary
 	FileWrites    DynamicAnalysisFileWrites
+	URLs          []string
```

Comment:
`URLs` belongs in `StraceSummary` below so that it is recorded per-phase and is output correctly.

Without this, no results will be output.